scenarios/change-data-capture/README.md

```diff
@@ -0,0 +1,171 @@
+# Pravega CDC Demo
+
+This page demonstrates how to preserve the CDC (Change Data Capture) from Mysql into Pravega and consume them in Flink.
+
+## Preparation
+
+The demo runs under `docker-compose`, so a Linux or Windows (WSL2) environment is required.
+
+With `docker` enabled, download the [`docker-compose.yml`](https://raw.githubusercontent.com/thekingofcity/Pravega-CDC-demo/master/docker-compose.yml) and put it anywhere you like.
+
+After having the `docker-compose.yml`, start it like this:
+
+```bash
+docker-compose up -d
+```
+
+This command automatically starts all the containers defined in the docker compose configuration in a detached mode. Run `docker ps` to check whether the 7 containers are running properly. You can also visit `http://localhost:12081/` to see if Flink is running normally.
+
+The docker compose environment consists of the following containers:
+
+- Flink SQL CLI: used to submit queries and visualize their results.
+- Flink Cluster: a Flink JobManager and a Flink TaskManager container to execute queries.
+- MySQL: MySQL 8 with some pre-populated tables in the database.
+- Pravega: preserve the Mysql change data.
+- Debezium: capture the Mysql change data and sink them to Pravega.
+- DataGen: the data generator. After the container is started, stock data from previous 5 days are pulled and updated to the Mysql. There were more than 1800 data points which should run at least half an hour.
```

Comment:
It looks confusing with both 5 days and half an hour if they do not know about the speedup we set. I think we can rephrase it in a more general way, like speeding up the changes of 5 days into half an hour.